requirements_dev.txt

```diff
@@ -14,5 +14,7 @@ SQLAlchemy>=1.4
 appdirs
 typing_extensions>=3.7
 SQLAlchemy_Utils>=0.37.8
-
+orjson
```

Comment:
Why?

Reply:
It was missing in the requirements_dev.txt since I added it when speeding up import/export.